stdlib/opcode.pyi

```diff
@@ -1,6 +1,23 @@
 import sys
 from typing_extensions import Literal
 
+__all__ = [
+    "cmp_op",
+    "hasconst",
+    "hasname",
+    "hasjrel",
+    "hasjabs",
+    "haslocal",
+    "hascompare",
+    "hasfree",
+    "opname",
+    "opmap",
+    "HAVE_ARGUMENT",
+    "EXTENDED_ARG",
+    "hasnargs",
+    "stack_effect",
```

Comment:
`stack_effect` is only available on 3.8+.

Reply:
I am able to use it on `3.6`:

```python
» ~/.pyenv/versions/3.6.12/bin/python
Python 3.6.12 (default, Feb 11 2021, 15:34:49) 
[GCC 4.2.1 Compatible Apple LLVM 11.0.0 (clang-1100.0.33.16)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import dis
>>> dis.stack_effect
<built-in function stack_effect>
>>> import opcode
>>> opcode.stack_effect
<built-in function stack_effect>
>>> from opcode import *
>>> stack_effect
<built-in function stack_effect>
```

Reply:
Then the if statement at the end of this file is wrong :)

Reply:
Alright, nevermind. GitHub only showed the last 5 lines or so after I clicked a thing.